devmode.sh

```diff
@@ -7,7 +7,7 @@ echo "## "
 if [ ! -f .devmode ]; then
   echo "## It looks like this is your first time using devmode. Lets gather some prerequisites..." && sleep 1
   mix deps.get
-  cd assets && yarn
+  cd assets && yarn && yarn deploy
```

Comment:
I don't think yarn deploy is actually necessary here, these assets will be built when `mix phx.server` is run

Reply:
They weren't accessible for me after I followed the start-up guide. I tried it once with mostly automatic and once with mostly manual setup. Each time I got an unstyled login page and a nice 404 instead of `app.css`. Once I ran `yarn build` and restarted the server - it was 👌.

Could it be that the server doesn't serve static assets from directories that weren't present on start-up? It finishes spinning up while frontend stuff is still building, creates an index of static assets before they're all there and then ignores requests for them? If that's the case, it would have been enough for me to simply restart the server 🤔. I'll check.

Reply:
@bitbldr 
Alright, so reloading the page does nothing, but restarting the server helps (which lends some credibility to my theory about static assets), but I think I prefer running `yarn build` to telling people to run `mix phx.server`, kill it and then run `mix phx.server` again ;)

Reply:
Here's the error (when you run the server for the first time):
![Screenshot 2023-11-28 at 18 46 00](https://github.com/Simon-Initiative/oli-torus/assets/2022097/2e49337b-44e6-4544-b6fc-056e4435362a)

And here's the login screen in all its unstyled glory:
![Screenshot 2023-11-28 at 18 47 41](https://github.com/Simon-Initiative/oli-torus/assets/2022097/b2f2763c-aa0a-49d9-80c2-397873fd7607)